src/app/features/Wallet/wallet/wallet.component.html

```
@@ -0,0 +1,47 @@
+<div class="max-w-5xl mx-auto p-6 space-y-6">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)